streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -1867,7 +1849,6 @@ void maybeInitTaskTimeoutsOrThrow(
         for (final TopicPartition partition : partitions) {
             final Task task = getActiveTask(partition);
             task.maybeInitTaskTimeoutOrThrow(nowMs, timeoutException);
```

Comment:
```suggestion
getActiveTask(partition).maybeInitTaskTimeoutOrThrow(nowMs, timeoutException);
```
coupling now seems obsolete, considering cohesion.